getinsomnia/insomnia
# feat: enable Cookie in pre-request scripting [INS-3379]

Changes:
- [x] enable Cookie in pre-request scripting


### How to test:

Please run this script and check its output.
```
const { Cookie } = require('insomnia-collection');

	const cookie1Opt = {
			key: 'myCookie',
			value: 'myCookie',
			expires: '01 Jan 1970 00:00:01 GMT',
			maxAge: '7',
			domain: 'domain.com',
			path: '/',
			secure: true,
			httpOnly: true,
			hostOnly: true,
			session: true,
			extensions: [{ key: 'Ext', value: 'ExtValue' }],
	};

const cookie1 = new Cookie(cookie1Opt);
console.log(cookie1.toString());
```
